pom.xml

```diff
@@ -161,6 +161,12 @@
       <groupId>org.apache.maven.plugins</groupId>
       <artifactId>maven-dependency-plugin</artifactId>
       <version>3.4.0</version>
+      <exclusions>
+        <exclusion>
+          <groupId>asm</groupId>
+          <artifactId>asm</artifactId>
+        </exclusion>
+      </exclusions>
```

Comment:
Was pulling in an ancient version of ASM 3, which is not compatible with Jetty Maven plugin.